lib/exq_ui_web/live/components/nav_bar_component.html.heex

```diff
@@ -4,26 +4,55 @@
     <div class="collapse navbar-collapse">
       <ul class="navbar-nav me-auto mb-lg-0">
         <li class="nav-item">
-          <%= nav_link(@socket, "Dashboard", Routes.dashboard_path(@socket)) %>
+          <.link
+            navigate={Routes.dashboard_path(@socket)}
+            class={nav_link_class(@socket, "Dashboard")}
+          >
+            Dashboard
+          </.link>
         </li>
         <li class="nav-item">
-          <%= nav_link(@socket, "Busy", Routes.busy_index_path(@socket)) %>
+          <.link navigate={Routes.dashboard_path(@socket)} class={nav_link_class(@socket, "Busy")}>
+            Busy
+          </.link>
         </li>
```

Comment:
because I can't render ~H in the helper classes, I split out the nav_link helper to provide the class but we have to use the ./link here